Doc/library/subprocess.rst

```diff
@@ -894,14 +894,16 @@ The following attributes are also available:
 
 .. attribute:: Popen.returncode
 
-   The child return code, set by :meth:`poll` and :meth:`wait` (and indirectly
-   by :meth:`communicate`).  A ``None`` value indicates that the process
+   The child return code. The child return code. Initially ``None``, :attr:`returncode` is set by :meth:`poll` and :meth:`wait` after process termination is observed.
```

Comment:
The docs failure is probably due to this line being too long - can you wrap it and keep it no more than 80 chars?

Reply:
I have addressed this issue.